language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/ImplementAllCodeAction.java

```diff
@@ -62,6 +63,7 @@ public List<CodeAction> getCodeActions(CodeActionContext context, RangeBasedPosi
 
         if (posDetails.matchedCodeActionNode().kind() != SyntaxKind.CLASS_DEFINITION
                 && posDetails.matchedCodeActionNode().kind() != SyntaxKind.OBJECT_METHOD_DEFINITION
+                && posDetails.matchedCodeActionNode().kind() != SyntaxKind.OBJECT_CONSTRUCTOR
```

Comment:
Shall we include these checks in the validate method. And let's add a negative test case covering this

Reply:
This case is already covered in `implementAllInObjConstructorExpr1.json` and `implementAllInObjConstructorExpr2.json`.